govcd/vm_test.go

```diff
@@ -967,23 +719,27 @@ func (vcd *TestVCD) Test_VMSetGetGuestCustomizationSection(check *C) {
 	guestCustomizationPropertyTester(vcd, check, vm)
 }
 
-// Test create internal disk For VM
-func (vcd *TestVCD) Test_AddInternalDisk(check *C) {
+// Test create/update/remove of Internal Disk
+func (vcd *TestVCD) Test_InternalDisk(check *C) {
 	fmt.Printf("Running: %s\n", check.TestName())
 
 	// In general VM internal disks works with Org users, but due we need change VDC fast provisioning value, we have to be sys admins
 	if vcd.skipAdminTests {
 		check.Skip(fmt.Sprintf(TestRequiresSysAdminPrivileges, check.TestName()))
 	}
-
-	vmName := "Test_AddInternalDisk"
-
+	vmName := check.TestName()
 	vm, storageProfile, diskSettings, diskId, previousProvisioningValue, err := vcd.createInternalDisk(check, vmName, 1)
 	check.Assert(err, IsNil)
 
+	// verify VM description still available - test for bugfix #418
```

Comment:
This comment should be deleted. There is no description check in the code below

Reply:
Removed.